prototypes/basic/app/views/select_header_row.html

```diff
@@ -19,14 +40,13 @@
             <h1 class="govuk-heading-l">
                 Select cells containing column names
             </h1>
-            <p>
+            <p class="govuk-body-l">
                 Column names are the descriptive labels usually found at the top
-                of a column.
-            </p>
-            <p>
-                You can select a whole row or just the column names that are
+                of a column. You can select a whole row or just the column names that are
                 needed for this service.
             </p>
+            <p class="govuk-body">
+                If you don't choose a column name,
```

Comment:
This looks incomplete?

Reply:
Oh yeah, went looking for what happened and got distracted :(

Reply:
Improved number parsing in 307045c

It might be worth us investing some time later on in a decent type parser to accurarely identify types given a bytestream and a set of options (e.g. how strict the parsing should be) as it would be useful both for this and in type validation as well as other parts of the future system.